ably/rest/auth.py

```diff
@@ -154,7 +157,7 @@ def request_token(self, key_name=None, key_secret=None, query_time=None,
         return TokenDetails.from_dict(response_json)
 
     def create_token_request(self, key_name=None, key_secret=None,
-                             query_time=False, token_params=None):
+                             query_time=None, token_params=None):
```

Comment:
This is more idiomatic than having a class created for that. And having to change the calls can be avoided by calling it with named arguments.